graspy/match/faq.py

```diff
@@ -0,0 +1,168 @@
+# Copyright 2019 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import math
+from scipy.optimize import linear_sum_assignment
+from scipy.optimize import minimize_scalar
+from .skp import SinkhornKnopp
+
+
+class FastApproximateQAP:
```

Comment:
modify the documentation file such that sphinx will render your documentation, and then you can prieview it on netifly. lets talk more about this tomorrow if that doesnt make sense